packages/cli/src/rest/api.ts

```diff
@@ -33,7 +34,34 @@ export function getDefaults () {
   const baseURL = environments[env].apiUrl
   const Authorization = `Bearer ${apiKey}`
 
-  return { baseURL, accountId, Authorization }
+  return { baseURL, accountId, Authorization, apiKey }
+}
+
+export async function isAuthenticated (): Promise<boolean> {
+  if (!config.hasValidCredentials()) {
+    throw new Error('Run `npx checkly login` or manually set `CHECKLY_API_KEY` ' +
+      '& `CHECKLY_ACCOUNT_ID` environment variables to setup authentication.')
+  }
+
+  const accountId = config.getAccountId()
+  const apiKey = config.getApiKey()
+
+  try {
+    // check if credentials works
+    await accounts.get(accountId)
+    return true
+  } catch (err: any) {
+    // if fails remove stored credentials
```

Comment:
I think that we can just leave the credentials. Probably the user will run another `npx checkly login` command and login again.

I'm just thinking that we should avoid taking a permanent action like wiping the auth config without the user explicitly requesting that. Maybe there was just a temporary bug that caused the 401 for example.